src/main/java/seedu/realodex/model/person/Birthday.java

```diff
@@ -16,14 +16,19 @@
  */
 //@@author 4llysa
 public class Birthday {
-
     public static final String INPUT_DATE_PATTERN = "ddMMMyyyy";
-    public static final String MESSAGE_CONSTRAINTS = "Birthday should be in " + INPUT_DATE_PATTERN + " format\n"
-            + "A valid date should be given, and it cannot be in the future and no earlier than year 1000.";
+    public static final String INPUT_MONTH_PATTERN = "MMM";
+    public static final String MESSAGE_CONSTRAINTS = "Birthday should be in " + "ddMMMyyyy" + " format.\n"
+            + "Date should also not be in future years and no earlier than year 1000!\n"
+            + "Example: b/17Sep2001";
```

Comment:
Good Example!

Reply:
Perhaps could consider making the message more clear bc i might think that the MMM should be a numerical value

Reply:
so like "where MMM is alphabetical"